.github/workflows/commit.yml

```diff
@@ -140,14 +141,15 @@ jobs:
           BRANCH="$(git branch --show-current)"
           git push --atomic origin "${BRANCH}" "${GIT_TAG}"
           echo "git_tag=${GIT_TAG}" >> "${GITHUB_OUTPUT}"
-  cut-release:
-    name: Cut a release
-    needs: [bump-version]
+  distribute:
+    name: Distribute
+    needs: [generate-matrixes, bump-version]
```

Comment:
This doesn't depend on `test` meaning that `test` doesn't need to run after a merge.  This allows us to build once post-merge.